### DISCUSSION TITLE
Explode-item flag in config.yml

### DISCUSSION THREAD

**Discussion by agoniaprivat** [PurpurMC/Purpur#1121] [Feature Requests]
Hey!

I want a explode-item flag in the config.yml for the endcrystal.

Example:

`explode-items: true/false`

Setting it to false, will make so items don't get destroyed.

**Why should you add this?**
There is a lot of pvp servers out there- and a lot of endcrystal pvpers. Also, Purpur is a great software for making your own gameplay 
unique.

**Comment by BillyGalbreath** [comment]
I'm sure this can be achieved with the existing end crystal settings.

https://purpurmc.org/docs/Configuration/end-crystal

The baseless end crystals are ones players place down. The ones with a base are generated when a dragon spawns.
